services/src/main/java/org/keycloak/services/clientpolicy/executor/SecureCibaSessionEnforceExecutor.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.services.clientpolicy.executor;
```

Comment:
Since this is CIBA specific executor, it will be ideal to add it under package `org.keycloak.protocol.oidc.grants.ciba.executor` to make the CIBA related stuff consolidated in single place within the codebase as much as possible. WDYT?

This comment applies to all executor and executorFactory classes.

Reply:
@mposolda I see, I will move this executor to `org.keycloak.protocol.oidc.grants.ciba.executor` as you've proposed.